package.json

```diff
@@ -77,6 +76,7 @@
     "babel-loader": "8.2.5",
     "babel-plugin-istanbul": "^6.1.1",
     "babel-plugin-transform-react-remove-prop-types": "^0.4.24",
+    "caniuse-lite": "^1.0.30001373",
```

Comment:
So caniuse-lite is the new @bundle-analyzer/webpack-plugin, right? Because it's more lightweight.

Reply:
Can `caniuse-lite` is a database that Babel with `babel/preset-env` uses during the Webpack build process.